Events/src/test/java/com/github/swcraftlyon/meetup/domain/EventManagerTest.java

```diff
@@ -0,0 +1,22 @@
+package com.github.swcraftlyon.meetup.domain;
+
+import com.github.swcraftlyon.meetup.domain.entities.Event;
+import com.github.swcraftlyon.meetup.domain.services.EventManager;
+import com.github.swcraftlyon.meetup.domain.services.IEventManager;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+public class EventManagerTest {
+
+    @Test
+    public void shouldCreateAnEvent() throws Exception {
```

Comment:
les méthodes de test doivent avoir la forme `should...when...`

le `when` est très important.